aten/src/ATen/native/native_functions.yaml

```diff
@@ -3322,8 +3322,8 @@
 - func: atan2_(Tensor(a!) self, Tensor other) -> Tensor(a!)
   variants: method
   dispatch:
-    CPU: legacy::cpu::_th_atan2_
-    CUDA: legacy::cuda::_th_atan2_
+    CPU: atan2_
```

Comment:
No need to define dispatch if all function names matches.

Reply:
Fixed